concrete/js/build/core/app/in-context-menu.js

```diff
@@ -259,7 +259,7 @@
                 my.hide(e);
             });
 
-            $(document).unbind('.concreteMenuDisable').on('click.concreteMenuDisable', function (e) {
+            $(document).unbind('.concreteMenuDisable').on('click.concreteMenuDisable','*', function (e) {
```

Comment:
maybe attach the "concreteMenuDisable" event to $(document.body)  ?

Reply:
@dimger I agree: in the past I've solved similar problems this way